### PR TITLE
Networking mappings

### PR DESCRIPTION
**Pull request by TheGlitch76**
I mostly focused on mapping around the new Configuration feature; this isn't a complete mapping of the network packages, but it gets us pretty close.

These mappings will help with the QSL networking/reg sync port

### CODE REVIEW

**Comment by TheGlitch76** [comment]
This is ready for review now.

**Comment by eristhea** [comment]
javadoc error: 
```
/net/minecraft/client/network/ServerInfo.java:275: error: reference not found
    * net.minecraft.network.packet.s2c.play.ResourcePackS2CPacket server
```

**Comment by TheGlitch76** [comment]
love that it doesn't show up from `gradlew build`

File: mappings/net/minecraft/client/network/AbstractClientNetworkHandler.mapping [hunk 1]
```diff
@@ -0,0 +1,46 @@
+CLASS net/minecraft/unmapped/C_xcfjcqcn net/minecraft/client/network/AbstractClientNetworkHandler
+	FIELD f_epofvvju serverInfo Lnet/minecraft/unmapped/C_xotldzgg;
+	FIELD f_ixguppmf connection Lnet/minecraft/unmapped/C_oasmcckk;
+	FIELD f_kjyjkrbj deferredPackets Ljava/util/List;
+	FIELD f_lhgeaaaz serverBrand Ljava/lang/String;
+	FIELD f_qcgabfsl CONNECTION_LOST_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/network/AbstractClientNetworkHandler.mapping]:
```suggestion
	FIELD f_qcgabfsl CONNECTION_LOST Lnet/minecraft/unmapped/C_rdaqiwdt;
```

File: mappings/net/minecraft/server/network/ServerChunkSender.mapping [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		COMMENT Drops the chunk from the queue (if not yet sent), otherwise tells the player to unload the chunk via {@link net.minecraft.network.packet.s2c.play.ChunkUnloadS2CPacket}.
+		ARG 1 player
+		ARG 2 pos
+	METHOD m_luffamtr collectChunksToSend (Lnet/minecraft/unmapped/C_gxrpzacc;Lnet/minecraft/unmapped/C_ynrszrtu;)Ljava/util/List;
+		ARG 1 chunkManager
+		ARG 2 pos
+	METHOD m_rsmbjmmj onChunkBatchAcknowledged (F)V
+		ARG 1 desiredChunksPerTick
+	METHOD m_uwdzfiae isChunkPending (J)Z
+		ARG 1 posLong
```
Comment (eristhea) [inline mappings/net/minecraft/server/network/ServerChunkSender.mapping]:
```suggestion
		ARG 1 pos
```
long positions are always just `pos`, we're kinda assuming that if you're touching chunk internals you know about long encoding for positions

File: mappings/net/minecraft/network/listener/C2SPacketListener.mapping [hunk 3]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_lpmetgbe net/minecraft/network/listener/C2SPacketListener
```
Comment (eristhea) [inline mappings/net/minecraft/network/listener/C2SPacketListener.mapping]: This name is contradictory to existing names:
A matching class for `S2C` is named `ClientPacketListener`, solving the "we already have that just below it in the inheritance tree" problem by renaming the other class `ClientPacketListener` -> `ClientCommonPacketListener`. I'm open to criticism on this decision (it was mine) but I think this is the better choice.

**Review by eristhea — CHANGES_REQUESTED** [review]

**Comment by eristhea** [comment]
> love that it doesn't show up from `gradlew build`

`gradle build javadocJar` baybee

File: mappings/net/minecraft/client/network/AbstractClientNetworkHandler.mapping [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	METHOD <init> (Lnet/minecraft/unmapped/C_ayfeobid;Lnet/minecraft/unmapped/C_oasmcckk;Lnet/minecraft/unmapped/C_aswgqekn;)V
+		ARG 2 connection
+		ARG 3 connectionState
+	METHOD m_dsehahmy (Lnet/minecraft/unmapped/C_wrmtlwqx;Ljava/net/URL;Ljava/lang/String;ZZ)V
+		ARG 5 accepted
+	METHOD m_fhwlneml sendWhen (Lnet/minecraft/unmapped/C_zyjtrjrl;Ljava/util/function/BooleanSupplier;Ljava/time/Duration;)V
+		ARG 1 packet
+		ARG 2 sendCondition
+		ARG 3 expirationTime
+	METHOD m_gmmwbpns preparePackPrompt (Lnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_rdaqiwdt;)Lnet/minecraft/unmapped/C_rdaqiwdt;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/network/AbstractClientNetworkHandler.mapping]:
```suggestion
	METHOD m_gmmwbpns composePackPrompt (Lnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_rdaqiwdt;)Lnet/minecraft/unmapped/C_rdaqiwdt;
```
?
`compose` seems like a more appropriate verb for combining `Text`s.
I think `prepare` has implications that timing or future changes are important.

File: mappings/net/minecraft/client/network/AbstractClientNetworkHandler.mapping [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+		ARG 0 info
+		ARG 1 sentPrompt
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/network/AbstractClientNetworkHandler.mapping]:
```suggestion
		ARG 1 prompt
```
it hasn't been sent yet, and the value passed is a param named `prompt`

File: mappings/net/minecraft/network/ServerConfigurationPacketHandler.mapping [hunk 6]
```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/unmapped/C_rvjamuad net/minecraft/network/ServerConfigurationPacketHandler
+	FIELD f_cjvcmoaq playerProfile Lcom/mojang/authlib/GameProfile;
+	FIELD f_cqvpsrsi tasks Ljava/util/Queue;
+	FIELD f_dfyozhxh INVALID_PLAYER_DATA_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/network/ServerConfigurationPacketHandler.mapping]:
```suggestion
	FIELD f_dfyozhxh INVALID_PLAYER_DATA Lnet/minecraft/unmapped/C_rdaqiwdt;
```

File: mappings/net/minecraft/network/listener/AbstractServerPacketHandler.mapping [hunk 7]
```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/unmapped/C_uhmrlzli net/minecraft/network/listener/AbstractServerPacketHandler
+	FIELD f_akpbpalo TIMEOUT_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/network/listener/AbstractServerPacketHandler.mapping]:
```suggestion
	FIELD f_akpbpalo TIMEOUT Lnet/minecraft/unmapped/C_rdaqiwdt;
```

File: mappings/net/minecraft/server/network/ServerLoginNetworkHandler.mapping [hunk 8]
```diff
@@ -19,6 +19,7 @@ CLASS net/minecraft/unmapped/C_rktsiyfy net/minecraft/server/network/ServerLogin
 	FIELD f_kankoaiw serverId Ljava/lang/String;
 	FIELD f_krozceue nextAuthenticatorThreadId Ljava/util/concurrent/atomic/AtomicInteger;
 	FIELD f_qcawoghx TIMEOUT_TICKS I
+	FIELD f_qhjiabpa UNEXPECTED_RESPONSE_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/server/network/ServerLoginNetworkHandler.mapping]:
```suggestion
	FIELD f_qhjiabpa UNEXPECTED_RESPONSE Lnet/minecraft/unmapped/C_rdaqiwdt;
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Comment by eristhea** [comment]
@TheGlitch76 can I polish this up for you, or would you like to come back to it?

**Comment by TheGlitch76** [comment]
Please do!

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by eristhea — APPROVED** [review]

**Pull request merged**